Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.m

```diff
@@ -113,46 +112,40 @@
         elseif  ~isempty(strfind(scan_name,'M0'))
             [nii_files,ASLContext] = xASL_bids_MergeNifti(nii_files, 'M0');
         end
-    end    
+    end
+    
+    %% FME (Hadamard) Check
+    if numel(nii_files)>=1
+        [resultPath, resultFile] = xASL_fileparts(nii_files{1});
+        if exist(fullfile(resultPath, [resultFile '.json']), 'file')
+            resultJSON = spm_jsonread(fullfile(resultPath, [resultFile '.json']));
+            if isfield(resultJSON,'SeriesDescription')
```

Comment:
@jan-petr Also here same as above, very specific. Is fine, but it should be noted that this only for Siemens etc. If someone now tries to load a Philips Hadamard this can go wrong